package.json

```diff
@@ -1,29 +1,36 @@
 {
   "name": "paack-ui",
   "version": "1.0.0",
-  "description": "",
+  "description": "Ready-to-use components for Elm UI",
   "main": "index.js",
   "directories": {
     "doc": "docs"
   },
   "scripts": {
-    "watch-docs-old": "chokidar \"**/*.elm\" -c \"elm make docs/src/Main.elm\"",
-    "watch-docs": "parcel docs/public/index.html",
-    "build-docs": "parcel build docs/public/index.html",
-    "test": "elm-test",
-    "analyse": "elm-analyse"
+    "analyse": "elm-analyse",
+    "docs": "elm make --docs=dist/docs.json",
```

Comment:
why use elm make instead of parcel? this way we lose the local server and the hot reload.

Reply:
That is for previewing the docs, not related to the showcase